<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>281</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Choose boudrate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget">
            <item>
             <property name="text">
              <string>Channel 0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Channel 1</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Choose channel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget_2">
              <item>
               <property name="text">
                <string>125 kbit</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>500 kbit</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1000 kbit</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>11 bit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>29 bit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_3">
            <property name="text">
             <string>11/29 bit</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7"/>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>Listen mod</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="Connect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
